config/crd/bases/config.ratify.deislabs.io_verifiers.yaml

```diff
@@ -48,6 +48,9 @@ spec:
                 description: Parameters for this verifier
                 type: object
                 x-kubernetes-preserve-unknown-fields: true
+              source:
```

Comment:
when reviewing the walkthrough, @jimmyraywv mentioned concern about container  Immutability. If the CRD manager detect source changes, will the plugin be reloaded? is that a concern?

Reply:
jimmy shared also shared this [article](https://www.linkedin.com/posts/danlorenc_youtube-activity-7016387747762315265-EIC0?utm_source=share&utm_medium=member_desktop)

Reply:
Currently yes, the plugin is reloaded

So if you were to change your plugin source from `noelfdpo.azurecr.io/sample-plugin:v1` to `noelfdpo.azurecr.io/sample-plugin:v2`, the manager picks that up, runs through the download process again, and you get a new file on disk

The experience is similar to if you're using a tool like Terraform inside a container, and you change your module version

We discussed a bit on the community call, and this feature is behind a feature flag so that users can opt-in/out of using dynamic plugins